README.md

```diff
@@ -1,6 +1,6 @@
-# egghead.io Fix Common Git Mistakes Course Notes
+# egghead.io [Fix Common Git Mistakes Course Notes](https://egghead.io/courses/fix-common-git-mistakes)
 
-![](git-mistakes.png)
+<p align="center"><img src="git-mistakes.png" width="200"></p>
```

Comment:
HTML renders in markdown files so you can add an `img` tag like this and set the width how you like it. As well as use css to position the image

Reply:
never knew this, thank you!